### PR TITLE
workaround for resource cache invalidation

### PR DESCRIPTION
**Pull request by Rahien**

### CODE REVIEW

File: app/routes/mandatarissen/mandataris.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ export default class MandatarissenMandatarisRoute extends Route {
         'bekleedt.bestuursfunctie',
         'bekleedt.bevat-in',
         'beleidsdomein',
+        'status',
```
Comment (Rahien) [inline app/routes/mandatarissen/mandataris.js:39]: this ensures that when we refresh the model, these relations are also refreshed

File: app/utils/toasts.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ const notifyFormSavedSuccessfully = (toaster) => {
   toaster.notify('Het formulier werd correct opgeslagen.', 'Succes', {
     type: 'success',
     icon: 'circle-check',
+    timeOut: 5000,
```
Comment (Rahien) [inline app/utils/toasts.js:5]: got tired of seeing the success messages forever heaping up in the top right corner

File: app/components/mandatarissen/mandataris-edit-prompt.js [hunk 3]
```diff
@@ -59,7 +60,7 @@ export default class MandatenbeheerMandatarisEditPromptComponent extends Compone
       sourceGraph: SOURCE_GRAPH,
     });
 
-    setTimeout(() => this.args.mandataris.reload(), 1000);
+    setTimeout(() => this.router.refresh(), 1000);
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-edit-prompt.js:63]: refresh reloads all models up to this point, which is safer. It also means that if we request relations to be included those will be reloaded too

**Review by bfidlers — APPROVED** [review]
Looks good

**Pull request merged**